proto/neutron/interchainqueries/genesis.proto

```diff
@@ -8,57 +8,59 @@ import "neutron/interchainqueries/params.proto";
+  // KV Interchain Queries. Max amount of keys is limited by the module's `max_kv_query_keys_count`
+  // parameters.
   repeated KVKey keys = 4;
-
-  // The filter for transaction search ICQ
+  // A stringified list of filters for remote transactions search. Only applicable for the TX
+  // Interchain Queries. Example: "[{\"field\":\"tx.height\",\"op\":\"Gte\",\"value\":2644737}]".
+  // Supported operators: "eq", "lt", "gt", "lte", "gte". Max amount of filter conditions is limited
+  // by the module's `max_transactions_filters` parameters.
   string transactions_filter = 5;
-
-  // The IBC connection ID for getting ConsensusState to verify proofs
+  // The IBC connection ID to the remote chain (the source of querying data). Is used for getting
+  // ConsensusState from the respective IBC client to verify query result proofs.
   string connection_id = 6;
-
-  // Parameter that defines how often the query must be updated.
+  // Parameter that defines the minimal delay between consecutive query executions (i.e. the
+  // minimal delay between query results update).
   uint64 update_period = 7;
-
-  // The local chain last block height when the query result was updated.
+  // The local chain block height of the last query results update.
   uint64 last_submitted_result_local_height = 8;
-
-  // The remote chain last block height when the query result was updated.
+  // The remote chain block height that corresponds to the last query result update.
   ibc.core.client.v1.Height last_submitted_result_remote_height = 9;
-
-  // Amount of coins deposited for the query.
+  // Amount of coins paid for the Interchain Query registration. The deposit is paid back to the
+  // remover. The remover can be either the query owner (during the submit timeout) or anybody.
   repeated cosmos.base.v1beta1.Coin deposit = 10 [
     (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins",
     (gogoproto.nullable) = false
   ];
-
-  // Timeout before query becomes available for everybody to remove.
+  // The duration, measured in blocks, that must pass since the query's registration or its last
+  // result submission before the query becomes eligible for removal by anyone.
   uint64 submit_timeout = 11;
-
-  // The local chain height when the query was registered.
+  // The local chain block height of the Interchain Query registration.
   uint64 registered_at_height = 12;
 }
 
+// Represents a path to an IAVL storage node.
 message KVKey {
-  // Path (storage prefix) to the storage where you want to read value by key
-  // (usually name of cosmos-sdk module: 'staking', 'bank', etc.)
+  // The substore name used in an Interchain Query. Typically, this corresponds to the keeper's
+  // storeKey, usually the module's name, such as "bank", "staking", etc.
   string path = 1;
-  // Key you want to read from the storage
+  // A hexadecimal-encoded byte array representing the key for specific data in the module's storage.
```

Comment:
The same about _hexadecimal_ encoding: this is correct for only for one particular ABCI query, but here it's just bytes